CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+# OpenMEEG
+#
+# Copyright (c) INRIA 2005-2014. All rights reserved.
+# See LICENSE.txt for details.
+# 
+#  This software is distributed WITHOUT ANY WARRANTY; without even
+#  the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#  PURPOSE.
+
+cmake_minimum_required(VERSION 2.8.12)
+if (CMAKE_MAJOR_VERSION EQUAL 3)
+    cmake_policy(SET CMP0048 NEW)
+    cmake_policy(SET CMP0037 OLD)
+    set(PROJECT_VERSION VERSION 2.3.0.1)
+endif()
+
+# Set CMAKE_BUILD_TYPE to Release by default.
+
+set(build_type Release)
+if (DEFINED CMAKE_BUILD_TYPE)
+    set(build_type ${CMAKE_BUILD_TYPE})
+endif()
+set(CMAKE_BUILD_TYPE ${build_type} CACHE STRING 
+    "Choose the type of build, options are: None Debug Release RelWithDebInfo MinSizeRel.")
+
+set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake ${CMAKE_MODULE_PATH})
+
+project(OpenMEEG-superProject ${PROJECT_VERSION})
```

Comment:
Yes and no. The CMakeLists.txt in OpenMEEG already use this name. So I'm not sure on this one.